src/ray/object_manager/plasma/common.h

```diff
@@ -46,6 +46,12 @@ enum class ObjectState : int {
   PLASMA_SEALED = 2,
 };
 
+constexpr std::string_view corruptedRequestErrorMessage =
```

Comment:
```suggestion
constexpr std::string_view kCorruptedRequestErrorMessage =
```

constant naming convention